### ISSUE TITLE
Creating .env failed

### ISSUE THREAD

**Issue by ivancelot** [laravel-zero/laravel-zero#168]
Using version ^2.5 for vlucas/phpdotenv
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Nothing to install or update
Generating optimized autoload files
Require package via composer: ✔
Creating .env: failed
Creating .env.example: failed
Updating .gitignore: failed

**Comment by nunomaduro** [comment]
Are you using Laravel Zero on Mac or Linux?

**Comment by ivancelot** [comment]
Linux, the project has all permission

**Comment by nunomaduro** [comment]
The file `.env` got created?

**Comment by ivancelot** [comment]
Yea, sorry, but why the legend xD

**Comment by emmajiugo** [comment]
Hi,

The `.env` and `.env.example` files were not created on windows. `.ignore` file was not updated. But I checked the `composer.json` file and I discovered that it added the `vluca/phpdotenv` dependencies.

**Comment by emmajiugo** [comment]
I discovered this was timing out on 
```?[36m1   ?[39m?[33mSymfony\Component\Process\Process::checkTimeout()?[39m
      ?[32mC:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\symfony\process\Process.php?[39m:?[32m416?[39m```

I had to run it 4 times before it actually created.

**Comment by nunomaduro** [comment]
@emmajiugo Thanks for debugging this. Can you make a pull request to set a 0 timeout on the symfony process?

**Comment by emmajiugo** [comment]
@nunomaduro 
After installing the dotenv, I keep getting this below error anytime i run a command...Any help?

```
PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\FatalThrowableError
: Argument 1 passed to Dotenv\Dotenv::__construct() must be an instance of Doten
v\Loader, string given, called in C:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\l
aravel-zero\foundation\src\Illuminate\Foundation\Bootstrap\LoadEnvironmentVariab
les.php on line 28 in C:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\vlucas\phpdot
env\src\Dotenv.php:31
Stack trace:
#0 C:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\laravel-zero\foundation\src\Illu
minate\Foundation\Bootstrap\LoadEnvironmentVariables.php(28): Dotenv\Dotenv->__c
onstruct('C:\\Users\\Chigbo...', '.env')
#1 C:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\laravel-zero\framework\src\Boots
trap\LoadEnvironmentVariables.php(32): Illuminate\Foundation\Bootstrap\LoadEnvir
onmentVariables->bootstrap(Object(LaravelZero\Framework\Application))
#2 C:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\laravel-zero\foundation\src\Illu
minate\Foundation\Application.php(203): LaravelZero\Framework\Bootstrap\LoadEnvi
ronmentVariables in C:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\vlucas\phpdoten
v\src\Dotenv.php on line 31

Fatal error: Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Arg
ument 1 passed to Dotenv\Dotenv::__construct() must be an instance of Dotenv\Loa
der, string given, called in C:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\larave
l-zero\foundation\src\Illuminate\Foundation\Bootstrap\LoadEnvironmentVariables.p
hp on line 28 in C:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\vlucas\phpdotenv\s
rc\Dotenv.php:31
Stack trace:
#0 C:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\laravel-zero\foundation\src\Illu
minate\Foundation\Bootstrap\LoadEnvironmentVariables.php(28): Dotenv\Dotenv->__c
onstruct('C:\\Users\\Chigbo...', '.env')
#1 C:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\laravel-zero\framework\src\Boots
trap\LoadEnvironmentVariables.php(32): Illuminate\Foundation\Bootstrap\LoadEnvir
onmentVariables->bootstrap(Object(LaravelZero\Framework\Application))
#2 C:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\laravel-zero\foundation\src\Illu
minate\Foundation\Application.php(203): LaravelZero\Framework\Bootstrap\LoadEnvi
ronmentVariables in C:\Users\Chigbo_Ezejiugo\Desktop\rave\vendor\vlucas\phpdoten
v\src\Dotenv.php on line 31

```

**Comment by nunomaduro** [comment]
Sorry, my bad. The latest version should have that fixed: `v5.7.19`.
